.jenkins/build_docker_images.sh

```diff
@@ -22,5 +22,5 @@ for df in ${dockerfiles[@]}; do
    # Copy an archived version of the image to a public Google Storage Bucket
    docker save $container -o $tar_file
    gzip $tar_file
-   gsutil copy ${tar_file}.gz gs://vcm-jenkins/${tar_file}.gz
+   gsutil copy ${tar_file}.gz gs://vcm-ml-public/jenkins-tmp/${tar_file}.gz
```

Comment:
Can you name this something which doesn't include `-tmp` before merging? This is likely not going to be a temporary directory.

Reply:
Done. Using the dedicated Jenkins storage bucket now